src/redux/sagas/actions/removeVerifiedMembers.ts

```diff
@@ -32,13 +31,7 @@ function* removeVerifiedMembersAction({
   meta: { id: metaId, navigate, setTxHash },
   meta,
 }: Action<ActionTypes.ACTION_REMOVE_VERIFIED_MEMBERS>) {
-  const batchKey = 'removeVerifiedMembers';
-
-  const { removeVerifiedMembers, annotateRemoveVerifiedMembers } =
-    yield createTransactionChannels(metaId, [
-      'removeVerifiedMembers',
-      'annotateRemoveVerifiedMembers',
-    ]);
+  let txChannel;
```

Comment:
this entire file seems like a bad rebase, please revert it to the way it's on the feature branch